src/main/java/org/opentripplanner/middleware/aws/AwsApiGateway.java

```diff
@@ -0,0 +1,104 @@
+package org.opentripplanner.middleware.aws;
+
+import com.amazonaws.services.apigateway.AmazonApiGateway;
+import com.amazonaws.services.apigateway.AmazonApiGatewayClient;
+import com.amazonaws.services.apigateway.AmazonApiGatewayClientBuilder;
+import com.amazonaws.services.apigateway.model.CreateApiKeyRequest;
+import com.amazonaws.services.apigateway.model.CreateApiKeyResult;
+import com.amazonaws.services.apigateway.model.CreateUsagePlanKeyRequest;
+import com.amazonaws.services.apigateway.model.DeleteApiKeyRequest;
+import com.amazonaws.services.apigateway.model.GetUsagePlanRequest;
+import com.amazonaws.services.apigateway.model.GetUsagePlanResult;
+import com.amazonaws.services.apigateway.model.NotFoundException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsText;
+
+/**
+ * Manages all interactions with AWS api gateway.
```

Comment:
The stuff in LogController should probably be refactored to use methods in here.

Reply:
@landonreed - Agree. I wasn't sure when this should happen so added an item to the Refactor tasks "Consolidate AWS SDK integration"

Reply:
Stil LogController has been merged into dev I decided to update.